### PR TITLE
arelle: enable only on python 3.4 (merge as necessary)

### PR DESCRIPTION
**Pull request by roberth**
###### Motivation for this change

Python 3.4 is the version suggested by arelle.org
On other versions, the build fails, so this PR disables those.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by roberth** [comment]
This needs to be reworked because of #29435

**Comment by roberth** [comment]
***TLDR**: merge this if the root cause of the problem on py36 is not solved before release 17.09.*

Ok, the current situation is arelle on python3.6 works on the master branch, but for some reason, it does not build on the release branch. The cause of the problem seems to be outside the arelle build definition, considering that the arelle directory is the same in both branches.

If the problem causing the failure persists until the release, please merge this PR. Arelle is a 'niche' package that I have introduced into nixpkgs only recently. Normal users will use it on 3.4 ('official' version) anyway, because that's the definition of `pkgs.arelle`. Python 3.6 support is only relevant for developers who must use the latest Python for some reason. I don't expect this group of people to exist. If they do exist, they can use nixpkgs master instead. Very hypothetical. Please worry about relevant nixpkgs issues instead.
